articles/native-platforms/android/06-rules.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Rules
+description: This tutorial will show you how to add customized rules to your app.
+seo_alias: android
+---
+
+Rules are functions written in JavaScript that are executed in Auth0 as part of the transaction every time a user authenticates to your application. This way, you can enhance Auth0 capabilities customizing it up to your needs. To see a detailed description, please refer to [the full documentation](https://auth0.com/docs/rules).
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* AndroidStudio 2.0
+* Emulator - Nexus5X - Android 6.0 
+  :::
+
+
+### Before Starting
+
+Make sure you have completed either the [Login](01-login.md) or the [Custom Login](02-custom-login.md) examples.
+
+ 
+### 1. Create a Rule
+ 
+To create a rule, you just need to go to the [new rule page](https://manage.auth0.com/#/rules/new). From there, you can either use a predefined template or create one from scratch.
```

Comment:
@tenmaster Replace this `https://manage.auth0.com/#/rules/new`
with this `${uiURL}/#/rules/new`